tests/core.py

```diff
@@ -2878,13 +2903,6 @@ def test_get_connections_env_var(self):
         assert conns[0].password == 'password'
         assert conns[0].port == 5432
 
-    def test_get_connections_db(self):
```

Comment:
n00b qq: why this test get removed?

Reply:
This test is testing the defaile airflow_db convection. This did fail because localhost was on the test mysql or postgress. This connection is tested in all tests that does something with the meta database. Because of that removing was easier and valid.
But maybe I'm missing something here?